public/js/chat.js

```diff
@@ -38,6 +41,27 @@ var Chat = function(id, player) {
 		});
 	});
 
+	//pulling the emote json file from the dgg cdn, not sure how to do this without a cors proxy :(
+	$.get("https://cors-anywhere.herokuapp.com/https://cdn.destiny.gg/2.13.0/emotes/emotes.json", function(emoteList){
+		this.emoteList = emoteList;
```

Comment:
`this` here is in the context of the function that started one line above so probably should be `self`